### ISSUE TITLE
Bump coffeescript to 1.9.3

### ISSUE THREAD

**Issue by ccorcos** [meteor/meteor#4807]
***1 Upvote*** https://github.com/meteor/meteor/blob/devel/packages/coffeescript/package.js#L11

**Comment by anubhav94** [comment]
Hey @ccorcos,

Thanks for the feature request. We'll leave this open for now and get around to it soon. We need to ensure that the changes don't break any of Meteor's current functionality before we'll be able to bump the coffeescript version.

**Comment by ccorcos** [comment]
ok

**Comment by mjmasn** [comment]
+1 for updating...

Just hit a weird issue with 1.9.2 which was fixed in 1.9.3 in case anyone else comes across it:

```coffeescript
Meteor.methods
    "#{METHOD_PREFIX}/method1": ->
        return
    "#{METHOD_PREFIX}/method2": ->
        return
```

```
error: unexpected indentation
        "#{METHOD_PREFIX}/method1": ->
^^^^^^^^
```

Workaround is just assigning the methods to a variable then calling `Meteor.methods` with that variable, e.g.

```coffeescript
myMethods = 
    "#{METHOD_PREFIX}/method1": ->
        return
    "#{METHOD_PREFIX}/method2": ->
        return

Meteor.methods myMethods
```
